cmd.h

```diff
@@ -20,6 +22,7 @@ class Dictionary {
 private:
 	vector<Pair> system;
 	vector<FileDescriptor> fileDescriptorVector;
+	vector<string> commands = { "mkdir","exit","ls","cd","rm","touch" };
```

Comment:
az oke, hogy ezt hasznalod arra, hogy megallapitsd, hogy egy command valid-e, de utana ha jol sejtem, arra nem tudod hasznalni, hogy melyik commandhoz melyik fuggveny legyen meghivva. Innentol kezdve olyan nagyon sok ertelme nincs, mert gondolom ott van egy jo nagy if, aminek a vegere egy `else not valid`-dal ez megoldhato lenne. Illetve ha most van egy uj command, akkor ott is bele kell tenni egy uj else if-et, meg itt is ki kell egesziteni. Tehat ket helyen kell turkalni a kodot. Jol sejtem?